src/bucket.js

```diff
@@ -23,11 +24,14 @@ var Bucket = function(options) {
   assert(options.credentials, "credentials must be specified");
   assert(!options.bucketCDN || typeof(options.bucketCDN) === 'string',
          "Expected bucketCDN to be a hostname or empty string for none");
+  assert(options.monitor,     "options.monitor is required");
```

Comment:
nit, this error doesn't is a little different than the others... perhaps just "monitor must be given" ?  I'm using the bucket and credentials assertions above as an example.